### PR TITLE
Use the Kiota generated SDKs

### PR DESCRIPTION
**Pull request by andreaTP**

### CODE REVIEW

File: src/main/java/io/managed/services/test/client/kafkainstance/KafkaInstanceApiUtils.java [hunk 1]
```diff
@@ -54,19 +52,22 @@ public static KafkaInstanceApi kafkaInstanceApi(KafkaRequest kafka, String offli
...
-                    .build());
-        }
-
-        return new KafkaInstanceApi(client.setBasePath(uri), offlineToken);
+        var adapter = new OkHttpRequestAdapter(new BaseBearerTokenAuthenticationProvider(new RHAccessTokenProvider(offlineToken)));
+        adapter.setBaseUrl(uri);
+        ApiClient client = new ApiClient(adapter);
+
+//        TODO: should this be allowed?
+//        if (Environment.KAFKA_INSECURE_TLS) {
```
Comment (andreaTP) [inline src/main/java/io/managed/services/test/client/kafkainstance/KafkaInstanceApiUtils.java:60]: not sure what is going on here, should this be preserved?

**Comment by andreaTP** [comment]
Ready for review, or, at least, for verification! @henryZrncik @kornys 🙂

**Comment by kornys** [comment]
nice PR @andreaTP at least from code POV, @henryZrncik could you please verify that it works as expected? 
Our PR verify pipeline is not ready yet so we need to manual verification.

**Comment by henryZrncik** [comment]
IGNORE THIS BRANCH & PULL REQUEST (PURELY FOR DEBUGGING PURPOSE)

**Comment by henryZrncik** [comment]
re-closing in favour of https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/pull/502

**Pull request closed without merging**